core/src/main/java/org/akaza/openclinica/service/JobTriggerService.java

```diff
@@ -102,7 +102,7 @@ public void triggerJob(){
 
 					if(ruleSet.getItemId()!=null){
 						// item Specific Rule
-						System.out.println("*** Item Specific Rule ***");
+						logger.info("*** Item Specific Rule ***");
```

Comment:
any reason for setting this to info as opposed to debug?

Reply:
I have changed according to your comments